generators/client/templates/angular/src/main/webapp/scss/_global.scss

```diff
@@ -105,37 +29,12 @@ Logo styles
             width: 70px;
             display: inline-block;
             vertical-align: middle;
-            background: url("../images/logo-jhipster.png") no-repeat center center;
+            background: url("../content/images/logo-jhipster.png") no-repeat center center;
```

Comment:
no it's generated inside scss folder as well for me. So I had to fix this path to point to the images properly.